docs/contributing/codebase-overview.md

```diff
@@ -65,40 +66,45 @@ The source code is located in `/api/src` and the below folders are inside there.
-| `/random`                    | [@directus/random](https://github.com/directus/directus/tree/main/packages/random)                                       | Set of random-utilities for use in tests                     |
-| `/schema`                    | [@directus/schema](https://github.com/directus/directus/tree/main/packages/schema)                                       | Utility for extracting information about the database schema |
-| `/specs`                     | [@directus/specs](https://github.com/directus/directus/tree/main/packages/specs)                                         | OpenAPI Specification of the Directus API                    |
-| `/storage-driver-azure`      | [@directus/storage-driver-azure](https://github.com/directus/directus/tree/main/packages/storage-driver-azure)           | Azure file storage abstraction for `@directus/storage`       |
-| `/storage-driver-cloudinary` | [@directus/storage-driver-cloudinary](https://github.com/directus/directus/tree/main/packages/storage-driver-cloudinary) | Cloudinary file storage abstraction for `@directus/storage`  |
-| `/storage-driver-gcs`        | [@directus/storage-driver-gcs](https://github.com/directus/directus/tree/main/packages/storage-driver-gcs)               | GCS file storage abstraction for `@directus/storage`         |
-| `/storage-driver-local`      | [@directus/storage-driver-local](https://github.com/directus/directus/tree/main/packages/storage-driver-local)           | Local file storage abstraction for `@directus/storage`       |
-| `/storage-driver-s3`         | [@directus/storage-driver-s3](https://github.com/directus/directus/tree/main/packages/storage-driver-s3)                 | S3 file storage abstraction for `@directus/storage`          |
-| `/storage`                   | [@directus/storage](https://github.com/directus/directus/tree/main/packages/storage)                                     | Object storage abstraction layer for Directus                |
-| `/types`                     | [@directus/types](https://github.com/directus/directus/tree/main/packages/types)                                         | Shared types for Directus                                    |
-| `/update-check`              | [@directus/update-check](https://github.com/directus/directus/tree/main/packages/update-check)                           | Check if an update is available for a given package          |
-| `/utils`                     | [@directus/utils](https://github.com/directus/directus/tree/main/packages/utils)                                         | Utilities shared between the Directus packages               |
+| Package Name                                                                                                             | Description                                                                                                                      |
```

Comment:
Thanks for updating this table! However I wonder if we can make this a little different / more dynamic so we don't have to update this table every time something changes about the packages (which I'm also sure we will forge 🙈😄).

Reply:
Yes, that would be the goal for the TypeDocs. However, in the meantime this will need to be maintained until then. With us getting into a release cycle and low amount of PRs it is inconsequential to keep this updated and it is something that I can make sure stays updated in the meantime 😀